### DISCUSSION TITLE
Clarity for personal use license

### DISCUSSION THREAD

**Discussion by dragoonmc** [TimPaterson/Fusion360-Batch-Post#61] [Ideas]
I had an issue today because I mistakenly removed the m30 from the end gcode section for individual file structure as I believed it was tool change end not file end. 
I think a more detailed tool tip or a field header change would help prevent this for others.

**Comment by TimPaterson** [comment]
Here is the current tooltip for ending code sequence. M30 plays a starring role, so I'm not sure how to write it any clearer.

                "To combine operations generated individually, the ending sequence "
                "(which should only appear once) must be found. This entry is the "
                "list of G-codes that start this ending sequence. For example, M30 "
                "(end program) would normally be here, but it may not be the first "
                "G-code of the ending sequence. M5 (spindle stop), M9 (coolant "
                "stop) and G28/G30 (move home) are also candidates, but you should "
                "look at the code from your post processor to determine what "
                "will work in your case. Any one of the G-codes you enter here "
                "will mark the start of ending sequence."
